click-game/main.js

```diff
@@ -0,0 +1,33 @@
+//import Cookies from 'cookies'
+
+var button = document.getElementById('button');
+var clicksDiv = document.getElementById('clicks');
+var clickAdd = 1;
+var clicks = 0;
+
+clicksCookie = cookies.getCookie('clicks');
+if(clicksCookie != null || clicksCookie >= 1){
+    clicks = cookies.getCookie('clicks');
+}
+
+button.addEventListener('click', click);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['click' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/edazpotato/edazpotato.github.io/pullRequest?prid=4679772)